src/core/Dropdown/Dropdown/Dropdown.tsx

```diff
@@ -214,7 +214,7 @@ class BaseDropdown extends Component<DropdownProps & InnerRef> {
             <ListboxPopover
               position={positionMatchWidth}
               {...passDropdownPopoverProps}
-              onKeyDownCapture={(event: KeyboardEvent) => {
+              onKeyDownCapture={(event: React.KeyboardEvent) => {
```

Comment:
This could / should probably be explicitly imported from React, but there is probably no agreed way to use these types for the time being. We should consider aligning all of these.